app/controllers/TransferController.scala

```diff
@@ -111,7 +108,7 @@ class TransferController @Inject()(
         case CurrentAndPreviousYearsEligibility(false, Nil, _, _) =>
           throw new NoTaxYearsAvailable
         case CurrentAndPreviousYearsEligibility(false, previousYears, registrationInput, _)
-          if previousYears.nonEmpty => Ok(views.html.multiyear.transfer.previous_years(registrationInput, previousYears, false))
+          if previousYears.nonEmpty => Ok(views.html.multiyear.transfer.previous_years(registrationInput, previousYears, currentYearAvailable = false))
```

Comment:
Don't need to name currentYearAvailable here, there are only 3 params in this view.

Reply:
Ignore me, good practice to name boolean parameters